.palantir/revapi.yml

```diff
@@ -886,6 +886,13 @@ acceptedBreaks:
     - code: "java.field.serialVersionUIDChanged"
       new: "field org.apache.iceberg.util.SerializableMap<K, V>.serialVersionUID"
       justification: "Serialization is not be used"
+    - code: "java.method.exception.checkedAdded"
```

Comment:
we don't want to break existing APIs, so changes to this file need to be reverted

Reply:
> we don't want to break existing APIs, so changes to this file need to be reverted

It looks like `MetricsReporter` doesn't have to implement `Closeable`,  I'll fix it here.

Reply:
> we don't want to break existing APIs, so changes to this file need to be reverted

done.